use-case-guides/workday-request-time-off/README.md

```diff
@@ -208,150 +181,333 @@ curl --location 'https://{{domain}}.workday.com/ccx/api/absenceManagement/v1/{{i
-    if response.status_code == 200:
-        data = response.json()
-        # Check if we have at least one record in the data list
-        if data['total'] > 0 and len(data['data']) > 0:
-            return data['data'][0]['workdayID']
-        else:
-            raise HTTPException(status_code=404, detail="Worker ID not found.")
-    else:
-        raise HTTPException(status_code=response.status_code, detail="Failed to fetch worker ID.")
-
-# Endpoint to request time off
-@app.post("/request-time-off/")
-async def request_time_off(email: str, totype: str, start_date: str, end_date: str = None, comments: str = "", start_time: str = "", end_time: str = ""):
-    try:
-        access_token = get_access_token()
-        worker_id = get_worker_id_by_email(email, access_token)
-    except HTTPException as e:
-        return {"error": e.detail}
-    
-    workday_instance_id = "<Your_workday_instance_id>"
-    
-    url_request_off = f"https://wd2-impl-services1.workday.com/api/absenceManagement/v1/{workday_instance_id}/workers/{worker_id}/requestTimeOff"
-    headers = {
-        "Content-Type": "application/json",
-        "Authorization": f"Bearer {access_token}",
+    ```bash
+    curl --location 'https//{{base_url}}/workers/{{user.email_addr}}/time-off' \
+    --header 'Content-Type: application/json'
+    ```
+
+    ![API Connection](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-2.png)
+
+
+3. Test your `GET /workers/{email}/time-off` API to make sure it returns your employee's time off details as expected. You can configure the API for one employee only for testing purposes (like `user@workday.net` in the example below), and replace with the requester's email address before launching to production (with `{{user.email_addr}}`).
+
+    ![Test API](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-3.png)
+
+4. Save your connector after testing your API.
+
+5. Label your APIs with fields that you want to extract from the response. Set your identifier and description as your Time Off Type and Name. Click Next.
+
+    ![Label APIs](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-4.png)
+
+6. Add a Followup Action to Book Time Off.
+
+    * Type: `Ask clarifying questions and execute a solution`
+    * Name: `Book Time Off`
+
+    ![Add Followup Action](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-5.png)
+
+7. Configure slots from the query response to ensure you pass the time off type id from the query to the API action. You can view and add slots from the "🔑 View Slots" button in the "Design Conversation" step.
+
+    ![Configure Slots](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-6.png)
+
+8. Inside the "Design Conversation" step, ask what the starting date for your employee's time off request.
```

Comment:
"ask what the starting date" -> grammar issue

Reply:
fixed